src/Fusion.Summary.Api/Controllers/DepartmentsController.cs

```diff
@@ -43,31 +36,19 @@ public async Task<IActionResult> GetDepartmentsV1()
 
         #endregion Authorization
 
-        var ret = new List<ApiDepartment>();
+        var departments = await DispatchAsync(new GetAllDepartments());
```

Comment:
Unsure if I understand, but doesn't the GetAllDepartments query just return everything? Or are you thinking that we should add some filtering to it so that it does not return everything